From 784b8575ce2580f6c5a39c22d3dc2c8852f0b8a3 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Tue, 11 Mar 2025 14:44:10 +0100
Subject: [PATCH 1612/1695] drivers: remoteproc: transform some debug message
 in error

Transform debug message in error to output information on error not
only when the debug level is set.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 872acf57c6fe26cba21deead1f328bc2dafe14fa drivers: remoteproc: transform some debug message in error]
Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: I5d51b050c5be121e5129031b5bfe212b691087b5
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/443459
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 core/drivers/remoteproc/stm32_remoteproc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/drivers/remoteproc/stm32_remoteproc.c b/core/drivers/remoteproc/stm32_remoteproc.c
index 805b59568..c679b836f 100644
--- a/core/drivers/remoteproc/stm32_remoteproc.c
+++ b/core/drivers/remoteproc/stm32_remoteproc.c
@@ -565,7 +565,7 @@ TEE_Result stm32_rproc_set_boot_address(uint32_t rproc_id, paddr_t address)
 		return TEE_ERROR_BAD_PARAMETERS;
 
 	if (rproc->boot_addr) {
-		DMSG("Firmware boot address already set");
+		EMSG("Firmware boot address already set");
 		return TEE_ERROR_GENERIC;
 	}
 
@@ -582,7 +582,7 @@ TEE_Result stm32_rproc_enable_sec_boot(uint32_t rproc_id)
 		return TEE_ERROR_BAD_PARAMETERS;
 
 	if (rproc->tzen) {
-		DMSG("Firmware TrustZone already enabled");
+		EMSG("Firmware TrustZone already enabled");
 		return TEE_ERROR_GENERIC;
 	}
 
-- 
2.47.2

